circle.yml

```diff
@@ -16,7 +16,6 @@ dependencies:
     - sudo service apache2 restart
     - chmod a+rwx backup log public/rss upload
     - sudo composer self-update --no-interaction
-    - composer config --global --no-interaction github-oauth.github.com $GITHUB_OAUTH_TOKEN
```

Comment:
> Does CircleCI define an environment variable that we can use to make this line conditional?

Yup, it looks like there's a set of variables that are only defined for fork builds:
https://circleci.com/docs/environment-variables#building-pull-requests-that-come-from-forks

I'll try using one of these as the basis of a conditional.